src/frontend/src/components/images/ApiImage.tsx

```diff
@@ -1,6 +1,5 @@
 /**
- * Component for loading an image from the InvenTree server,
- * using the API's token authentication.
+ * Component for loading an image from the InvenTree server using the CSRF cookie
```

Comment:
If the browser session has this cookie now, do we still need the `ApiImage` component? Or could we replace it entirely with the `<Image>` component?

Reply:
Not sure, it is working on my deployment but not on netlify

Reply:
this seems resolved now